docs/reference/glossary.asciidoc

```diff
@@ -536,6 +542,13 @@ To grow the amount of shards in an index.
 See the {ref}/indices-split-index.html[split index API].
 // end::split-def[]
 
+[[glossary-system-index]] system index ::
+// tag::hidden-index-def[]
+An index that contains configuration information or other data used internally by the system, 
+such as the `.security` index
+You cannot directly access or modify system indices.
```

Comment:
I'm not sure if this is relevant because I'm not sure what branches this is aimed at, but this will only be true for 8.0+, for 7.x you can access/modify system indices, but doing so is deprecated.